packages/data/src/index.js

```diff
@@ -43,6 +43,7 @@ export { plugins };
  * The combineReducers helper function turns an object whose values are different
  * reducing functions into a single reducing function you can pass to registerReducer.
  *
+ * @type  {Function}
```

Comment:
Have you tried:

```
* @type {import('redux'). combineReducers}
```

Reply:
It's included in this file: https://unpkg.com/browse/redux@4.2.0/index.d.ts

Reply:
Good call!